### PR TITLE
✨add logly ad extension

### PR DESCRIPTION
**Pull request by RINDO**

### CODE REVIEW

**Comment by RINDO** [comment]
@googlebot We signed it!

**Comment by lannka** [comment]
@RINDO could you try to implement render-start API: https://github.com/ampproject/amphtml/blob/master/ads/README.md#other-tips

**Comment by RINDO** [comment]
@lannka fixed!! please review again😄

File: ads/logly.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const d = global.document.createElement('div');
+  d.id = 'logly-lift-' + data['adspotid'];
+  global.document.getElementById('c').appendChild(d);
+
+  const url =
+    'https://l.logly.co.jp/lift_widget.js' +
+    `?adspot_id=${encodeURIComponent(data['adspotid'])}`;
+
+  loadScript(global, url, () => {
+    window.context.renderStart();
```
Comment (lannka) [inline ads/logly.js]: you will need to call renderStart in your lift_widget.js when the ad is ready to be displayed

Comment (RINDO) [reply ads/logly.js]: fixed. I implemented `renderStart` and `noContentAvailable` in lift_widget.js 😃

**Pull request merged**